jpo-ode-svcs/src/main/resources/application.properties

```diff
@@ -7,23 +9,53 @@ spring.http.multipart.max-file-size=1MB
 spring.http.multipart.max-request-size=1MB
 
 #ode.kafkaBrokers = localhost:9092
+#ode.verboseJson = false
+#ode.externalIpv4 = <YOUR_EXTERNAL_IPV4 or define env variable $ODE_EXTERNAL_IPV4>
+#ode.externalIpv6 = <YOUR_EXTERNAL_IPV6 or define env variable $ODE_EXTERNAL_IPV6>
+
+#File import properties
 #ode.uploadLocation = uploads
 #ode.uploadLocationBsm = bsm
 #ode.uploadLocationMessageFrame = messageframe
-#ode.kafkaTopicBsm = j2735Bsm
-#ode.kafkaTopicBsmRawJson = j2735BsmRawJson
-#ode.kafkaTopicBsmFilteredJson = j2735BsmFilteredJson
+
+#USDOT Situation Data Clearinghouse (SDC)/ Situation Data Warehouse (SDW), a.k.a Data Distribution System (DDS) Properties
+#=========================================================================================================================
+
+#DDS WebSocket Properties
+#------------------------
 #ode.ddsCasUsername=<SDWUSERNAME> or define env variable $ODE_DDS_CAS_USERNAME
 #ode.ddsCasPassword=<SDWPASSWORD> or define env variable $ODE_DDS_CAS_PASSWORD
-# IPv4 address and listening port for SDC
+
+#IPv4 address and listening UDP port for SDC
+#-------------------------------------------
 #ode.sdcIp=104.130.170.234
 #ode.sdcPort=46753
 
-# the ip address where SDC will send the response back
-#ode.returnIp=<ODE IP ADDRESS> or define env variable $ODE_RETURN_IP
+#Enable/disable depositing sanitized BSMs to SDC
+#-----------------------------------------------
+#ode.depositSanitizedBsmToSdc=true
+
+#ode.serviceRespExpirationSeconds = 60
+
+#BSM Properties
+#--------------
+#ode.kafkaTopicBsm = j2735Bsm
+#ode.kafkaTopicBsmRawJson = j2735BsmRawJson
+#ode.kafkaTopicBsmFilteredJson = j2735BsmFilteredJson
+#ode.bsmReceiverPort = 46800
+#ode.bsmBufferSize = 500
+
+#Vehicle Situation Data (VSD) Properties
+#---------------------------------------
+#ode.kafkaTopicVsd = topic.J2735Vsd
+#ode.vsdReceiverPort = 46753
+#ode.vsdDepositorPort = 5555
+#ode.vsdBufferSize = 500
 
-#port to forward request to SDC, receive response from SDC and then forward response to Blackbox
-#ode.forwarderPort=5555
+#Intersection Situation Data (ISD) Properties
+#--------------------------------------------
+#ode.kafkaTopicEncodedIsd = topic.EncodedIsd
+#ode.isdReceiverPort = 46801

```

Comment:
Yes, currently we don't support same port numbers for these receivers. If we need to support that, we would need a main listener/dispatcher that determines what message is being received through the same port and dispatch it to the appropriate receiver.